packages/aws-cdk-lib/aws-apigateway/lib/authorizers/cognito.ts

```diff
@@ -78,7 +80,7 @@ export class CognitoUserPoolsAuthorizer extends Authorizer implements IAuthorize
       type: 'COGNITO_USER_POOLS',
       providerArns: props.cognitoUserPools.map(userPool => userPool.userPoolArn),
       authorizerResultTtlInSeconds: props.resultsCacheTtl?.toSeconds(),
-      identitySource: props.identitySource || 'method.request.header.Authorization',
+      identitySource: props.identitySource || IdentitySource.header('Authorization'),
```

Comment:
The change caused by this is guaranteed by [this test](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-apigateway/test/authorizers/cognito.test.ts#L27).